otter/utils.py

```diff
@@ -8,7 +8,7 @@
 import random
 import string
 import shutil
-
+import re
```

Comment:
A (super) nit-picky issue but since I'm starting a review anyway: can you move this import into the list of imports above in alphabetical order?